src/controllers/petsController.js

```diff
@@ -75,26 +75,44 @@ const getPets = async (req, res, next) => {
       log.debug("Calculates pet distances...");
       const { distance, zipCode } = req.query;
 
+      // only performs the calculation if a distance is specified
+      if (!distance) {
```

Comment:
I think we should only perform the calculation if both distance and zip code are specified. Currently, if someone selected "100 miles from" and no zip code, no pets will show up. I think it would make sense to show all animals instead

Reply:
makes sense